### ISSUE TITLE
[BUG] Translations don't show with repeater field

### ISSUE THREAD

**Issue by rolandow** [AnimalDesign/bolt-translate#90]
Translations seem to be saved correctly. I can see them in de admin interface of bolt, when I change the locales.

In the database though I see there is no json stored for nldata and nlslug, which are exactly the locales that do output information. The other locales don't.

I used this contenttype definition:

```yaml
carousel:
    name: Carousels
    singular_name: Carousel
    fields:
        name:
          type: text
        locale:
          type: locale
          group: content
        slug:
          type: slug
          uses: name
        nldata:
          type: hidden
        nlslug:
          type: locale_data
          index: true
        endata:
          type: hidden
        enslug:
          type: locale_data
          index: true
        dedata:
          type: hidden
        deslug:
          type: locale_data
          index: true
        items:
          type: repeater
          is_translateable: true
          fields:
              title:
                type: text
                class: large
              description:
                type: textarea
              image:
                type: image
```

I used this Twig snippet to display this content type:

```twig
<header id="myCarousel" class="carousel slide">
    {% setcontent carousel = "carousel/frontpage" %}

    {% if carousel is not empty and carousel.items is not empty and carousel.items is iterable %}
        <!-- Indicators -->
        <ol class="carousel-indicators">
            {% for item in carousel.items %}
                <li data-target="#carousel-example-generic" data-slide-to="{{ loop.index0 }}"{% if loop.first %} class="active"{% endif %}></li>
            {% endfor %}
        </ol>


        <!-- Wrapper for slides -->
        <div class="carousel-inner" role="listbox">
            {% for item in carousel.items %}
                <div class="item{% if loop.first %} active{% endif %}">
                    <div class="fill" style="background-image:url('{{ asset(item.image.value.file, "files") }}');"></div>

                    {#{{ item.image.value.file|showimage() }}#}
                    <div class="carousel-caption">
                        {% if item.title is not empty %}
                            <h1>{{ item.get('title') }}</h1>
                        {% endif %}
                        {% if item.description is not empty %}
                            {{ item.get('description') }}
                        {% endif %}
                    </div>
                </div>
            {% endfor %}
        </div>

        <!-- Controls -->
        <a class="left carousel-control" href="#myCarousel" data-slide="prev">
            <span class="icon-prev"></span>
        </a>
        <a class="right carousel-control" href="#myCarousel" data-slide="next">
            <span class="icon-next"></span>
        </a>

    {% endif %}
</header>
```

Details
-------

 - Translate Version: [ 4.0.1 ]
 - Install type: [ Extension Repository ]
 - Bolt Version: [ 3.2.3 ]
 - PHP version: [ 5.5.9-1ubuntu4.9 ]
 - Used webserver: [ Nginx ] version [ 1.4.6 ]

**Comment by madc** [comment]
Content of repeater fields is stored in a different table (`bolt_field_value`) for all contenttypes. Could you have a look, whats stored in there?

**Comment by madc** [comment]
I am able to recreate this issue and can confirm, that it still persists. Data are stored correctly, but hydrated differently for the default locale and the other locales.

Default:
![repeater_default](https://cloud.githubusercontent.com/assets/343392/21752610/1086489a-d5dc-11e6-8767-c507ab9ccdad.png)

Secondary:
![repeater_secondary](https://cloud.githubusercontent.com/assets/343392/21752611/138688a2-d5dc-11e6-8bcc-1bc4c638e357.png)

 So instead of accessing the data via `repeaterfield.fieldname`, it has to be accessed like `repeaterfield.0.value`.

**Comment by madc** [comment]
I've been discussing this with @SahAssar. This issue is based on one in the Bolt-core and we wont be able to fix it until the 4.0 release. The good news is, you can easily workaround this by using the get-notation (repeaterfield.get('fieldname')).

I'll add this info to the readme.
